### PR TITLE
Merge release-1.7.1 changes into release-1.8.0

### PR DESCRIPTION
**Pull request by VamsiSiddu-7**
# Description
This pr adds the release-1.7.1 csm-docs changes into release-1.8.0

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/885  |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Example: `samples/secret.yaml` for PowerFlex storage system v4.0.x
 ```
  *NOTE: To use multiple arrays, copy and paste section above for each array. Make sure isDefault is set to true for only one array.* 
 
-After editing the file, run the below command to create a secret called `vxflexos-config`. This assumes `vxflexos` is release name, but it can be modified during [install](/#install-the-driver):
+After editing the file, run the below command to create a secret called `vxflexos-config`. This assumes `vxflexos` is release name, but it can be modified during [install](../#install-the-driver):
```
Comment (sharont58) [inline content/docs/csidriver/installation/helm/powerflex.md:133]: ...run the command below ...

File: content/docs/csidriver/installation/operator/powerflex.md [hunk 2]
```diff
@@ -19,7 +20,7 @@ There are sample manifests provided which can be edited to do an easy installati
 Kubernetes Operators make it easy to deploy and manage the entire lifecycle of complex Kubernetes applications. Operators use Custom Resource Definitions (CRD) which represents the application and use custom controllers to manage them.
 
 ### Prerequisites:
-- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex.md) for details
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex) for details
```
Comment (sharont58) [inline content/docs/csidriver/installation/operator/powerflex.md:23]: ...volumes are blocked ...

(See Dell's inclusive language terminology at https://dell.sharepoint.com/sites/InclusiveLanguage_DellTechnologies/SitePages/Documents/DOC-474277.aspx)

File: content/v1/csidriver/installation/helm/powerflex.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ The following are requirements that must be met before installing the CSI Driver
 - If using Snapshot feature, satisfy all Volume Snapshot requirements
 - A user must exist on the array with a role _>= FrontEndConfigure_
 - If enabling CSM for Authorization, please refer to the [Authorization deployment steps](../../../../authorization/deployment/) first
-- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex.md) for details
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex/) for details
```
Comment (sharont58) [inline content/v1/csidriver/installation/helm/powerflex.md:32]: ... volumes are excluded by ...

(Inclusive language)

File: content/v3/csidriver/installation/helm/powerflex.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The following are requirements that must be met before installing the CSI Driver
 - If using Snapshot feature, satisfy all Volume Snapshot requirements
 - A user must exist on the array with a role _>= FrontEndConfigure_
 - If enabling CSM for Authorization, please refer to the [Authorization deployment steps](../../../../authorization/deployment/) first
-- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex.md) for details
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../troubleshooting/powerflex/) for details
```
Comment (sharont58) [inline content/v3/csidriver/installation/helm/powerflex.md:32]: ... volumes are excluded from multipathd...

**Review by sharont58 — CHANGES_REQUESTED** [review]
See comments

**Comment by VamsiSiddu-7** [comment]
Closing this pr I will raise a new pr Thanks

**Pull request closed without merging**